src/containers/CareerDetails.js

```diff
@@ -1,19 +1,123 @@
 import React, { Component } from 'react';
-import styled from 'styled-components';
+import { connect } from 'react-redux';
+import {
+  Icon,
+  CareerLogo,
+  TitleWrapper,
+  Description,
+  TopSection,
+  CareerTagline,
+  TitleTagWrapper,
+  CareerContainer,
+  CareerTitle,
+  SectionTitle,
+  DetailSection,
+  SalaryStart,
+  Start,
+  SalaryTenYear,
+  TenYear,
+  TableContainer,
+  TableColumn,
+  GradeBagrut,
+  ReqBagrut,
+  UserBagrut,
+  GradePsyc,
+  ReqPsyc,
+  UserPsyc,
+  GradeTawjihi,
+  ReqTawjihi,
+  UserTawjihi,
+} from './../styles/details.js';
 
-const CareerContainer = styled.div`
-  background-color: red;
-`;
-const MainTitle = styled.h1``;
+import { fetchDetails } from '../actions/career_details';
 
 class CareerDetails extends Component {
+  componentDidMount() {
+    this.props.fetchDetails({ career_id: this.props.match.params.id });
+  }
+
+  showDetails = () => {
+    if (this.props.careerDetail.dataFetched) {
```

Comment:
Add a temporary `else` that just has a 'Loading career details' div in it. I don't like that `showdetails()` is just undefined until data comes in.